### ISSUE TITLE
Voiceover on Safari (MacOS) doesn't read out select options

### ISSUE THREAD

**Issue by JamieSK** [orchidjs/tom-select#478]
**Describe the bug**
Using Voiceover on Safari, a user doesn't hear the options in the select, makes the select all but unusable.

**To Reproduce**
1. https://tom-select.js.org/ on Safari/MacOS
2. Turn on Voiceover (cmd + F5)
3. Tab to the select example at the top of the page or click in it
4. Cycle through the options with the arrow keys

**Expected behavior**
The options to be read out in some way, any other browser (tried Firefox and Chrome) with Voiceover seems to work.

**Additional context**
 - OS: MacOs
 - Browser: Safari
 - Version: 15.6.1 (probably others too)

<!--
Thanks for taking the time to submit your bug report,
and don't forget to star our repo!

**Comment by oyejorge** [comment]
If it works everywhere else, should probably submit it to https://webkit.org/reporting-bugs/
